src/inheritAttributes.js

```diff
@@ -272,7 +272,12 @@ export const toRepresentations =
 export const toAdaptationSets = (mpdAttributes, mpdBaseUrls) => (period, periodIndex) => {
   const periodBaseUrls = buildBaseUrls(mpdBaseUrls, findChildren(period, 'BaseURL'));
   const periodAtt = parseAttributes(period);
-  const periodAttributes = merge(mpdAttributes, periodAtt, { periodIndex });
+  const periodAttributes = merge(mpdAttributes, { periodIndex });
+
+  if (periodAtt.duration) {
+    periodAttributes.periodDuration = periodAtt.duration;
+  }
```

Comment:
Looks like we're losing the rest of the `periodAtt`, which I don't think we intended

Reply:
This is intended, I'd rather pass along attributes that will be used. That's what caused the bug initially since `Period@duration` and `AdaptationSet@duration` have the same name but mean different things.

Reply:
I guess I meant if there's some other attribute that should be merged to topLevel. As an alternative, we could prepend the period attributes with `period`

Reply:
At the moment we actually don't even use Period@duration so I could remove this part entirely.